src/test/java/org/sagebionetworks/bridge/adherence/WeeklyAdherenceReportWorkerProcessorTest.java

```diff
@@ -305,6 +319,163 @@ public void limitsScanToProvidedAppsAndStudies() throws Exception {
         verify(mockWorkersApi, never()).getWeeklyAdherenceReportForWorker("app1", "study1b", "user1");
         verify(mockWorkersApi, never()).getWeeklyAdherenceReportForWorker("app2", "study2b", "user1");
     }
+    
+    @SuppressWarnings("unchecked")
+    @Test
+    public void onlyRunsOnTargetedHourOfDay() throws Exception {
```

Comment:
Also, all the test cases get their time zone from the study. Should we have a test case that gets the time zone from the request?

Reply:
Added this case